translations/stripes-authorization-components/en.json

```diff
@@ -8,6 +8,7 @@
   "columns.updatedBy": "Updated by",
   "assignedUsers": "Assigned users",
   "assignUnassign": "Assign/Unassign",
+  "assignUnassign.modal.label": "Select User ({tenant})",
```

Comment:
Should be "Select user ({tenant})". FOLIO uses case, i.e. only the first word is capitalized.